lib/rubocop/ast/node/ensure_node.rb

```diff
@@ -13,6 +13,13 @@ def body
         node_parts[1]
       end
 
+      # Returns the `rescue` node of the `ensure`, if present.
+      #
+      # @return [Node, nil] The `rescue` node.
+      def rescue_node
+        node_parts[0] if node_parts[0].rescue_type?
+      end
```

Comment:
@marcandre do you want me to do this as a separate PR, or create a separate changelog entry for it pointing to this PR?